src/runtime_impl.cpp

```diff
@@ -528,6 +530,88 @@ namespace hpx {
         cond.notify_all();                  // we're done now
     }
 
+    int runtime_impl::suspend()
+    {
+        std::uint32_t initial_num_localities = get_initial_num_localities();
+        if (initial_num_localities > 1)
```

Comment:
What is the rationale of this limitation? Wouldn't it be perfectly ok to suspend the runtime on one of the nodes at a time?

Reply:
Simply that I haven't tested it in those situations, and I did not think it would be useful. But you're right, it *should* be okay in those cases as well. Do you see a use case for it?

Reply:
Providing a migration path for distributed MPI+X applications.

Reply:
This does not have to be part of this PR, however.

Reply:
Ok, I'd like to keep it separate in that case so that it can go in with appropriate tests/examples showing how it would be used in that case (and to see if there are any corner cases to be aware of).